### ISSUE TITLE
[Request] kando-git

### ISSUE THREAD

**Issue by D3vil0p3r** [chaotic-aur/packages#3302]
### Package:

Custom package: https://github.com/chaotic-aur/packages/tree/main/kando-git

Reference:
https://aur.archlinux.org/packages/kando-bin

Upstream:
https://github.com/kando-menu/kando

### Purpose:

A pie menu for the desktop. It will be highly customizable and will allow you to create your own menus and actions.

Unlike FlyPie, it is compatible with all DE and WM

### Copyright:

MIT

**Comment by Technetium1** [comment]
Adding when Electron version is upgraded https://endoflife.date/electron

**Comment by D3vil0p3r** [comment]
> Adding when Electron version is upgraded https://endoflife.date/electron

Must it be upgraded by Kando dev or you mean Arch Electron package?

**Comment by xiota** [comment]
I would prefer upstream Kando dev update the project.  Better for everyone in terms of internet security.

**Comment by D3vil0p3r** [comment]
@Schneegans can you take care of Electron version on Kando?

**Comment by Schneegans** [comment]
Yeah, I'll look into it. Last time I checked, there were some serious regressions in the window management both on macOS and Linux/X11 on more recent versions of Electron. But maybe this has been resolved by now.

**Comment by D3vil0p3r** [comment]
@Schneegans any news?

**Comment by Schneegans** [comment]
Well, the `main` branch of Kando uses Electron 32 since two weeks now. The only issue I encountered so far is [this one](https://github.com/electron/electron/issues/43415). It is said to be fixed in Electron 33 which is about to be released in a week. So if there are no other regressions, the next version of Kando will use Electron 33, I hope.

**Comment by D3vil0p3r** [comment]
@Schneegans any news about integration with Electron 33?

**Comment by Schneegans** [comment]
Well, the issue has not been fixed, despite some folks claiming it to be fixed. So I'll investigate whether it has any negative performance implications. If so, I'll see if it is feasible to use different electron versions on the different platforms. Although I would prefer not to do this...

**Comment by D3vil0p3r** [comment]
Did you try to run Kando by `electron33 /usr/lib/kando/app env ELECTRON_OZONE_PLATFORM_HINT=auto` or `electron33 /usr/lib/kando/app env ELECTRON_OZONE_PLATFORM_HINT=auto --no-sandbox`?

**Comment by Schneegans** [comment]
Sorry, I am afraid I do not get your point. I downgraded Electron for Kando now to version 31 to fix this macOS issue linked above. This will be supported for several months from now on, so hopefully the bug in 32 and 33 will get resolved until then.

**Comment by xiota** [comment]
Added as a custom -git package that tracks extra/electron.  Electron 37 seems to work.

May be dropped if breaks and too difficult to fix or usage is too low to make it worthwhile, etc.

@Schneegans Feel free to refer packaging related issues, need for rebuild, etc back here.
